collections/ffs.go

```diff
@@ -8,31 +8,31 @@ import (
 )
 
 type FFSInstance struct {
-	BucketKey string `bson:"_id"`
-	FFSToken  string
-	Archives  Archives
+	BucketKey string   `bson:"_id"`
+	FFSToken  string   `bson:"ffs_token"`
```

Comment:
Adding tags here that will make the raw keys match the format of everything else, which is nice when using the `mongo` shell to explore data.

Reply:
Cool.